### DISCUSSION TITLE
2024-06-04(화)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#97] [데일리스크럼]
아래 내용을 복사해서 작성해주시면 됩니다.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일
- onboarding 페이지 UI 구현

### 해야할 일
- axios intercepter 쿠키에서 토큰꺼내기

### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- 코드잇 리액트 쿼리 강의 완료
- 리액트 쿼리로 작업 필요한 부분 아이디어 구상

### 해야할 일
- 헤더 + 레이아웃 최종 작업
- 홈 페이지 및 상품 목록 리스트

### 어려운 일
- 파이팅!!

**Comment by seolsis** [comment]
### 완료한 일
- 결제 동의 기능 구현 (checkbox)
### 해야할 일
- 금액/상품목록 부분 더미데이터 생성해서 불러오기 (결제 UI)
### 어려운 일
- 죙일피곤

**Comment by q45402sk** [comment]
### 완료한 일
share모달

### 해야할 일
zzim 컴포넌트,  drawer

### 어려운 일

**Comment by minyoung0503** [comment]
### 완료한 일

### 해야할 일
profileImgBadge 기본 이미지 변경
리뷰 전체보기 페이지 

### 어려운 일
